src/Microsoft.Health.Dicom.Client/DicomWebClient.Workitem.cs

```diff
@@ -61,27 +79,30 @@ public async Task<DicomWebAsyncEnumerableResponse<DicomDataset>> QueryWorkitemAs
 
         request.Headers.Accept.Add(DicomWebConstants.MediaTypeApplicationDicomJson);
 
-        var response = await HttpClient.SendAsync(request, cancellationToken).ConfigureAwait(false);
+        var response = await HttpClient.SendAsync(request, cancellationToken)
+            .ConfigureAwait(false);
 
-        await EnsureSuccessStatusCodeAsync(response).ConfigureAwait(false);
+        await EnsureSuccessStatusCodeAsync(response)
+            .ConfigureAwait(false);
 
         return new DicomWebAsyncEnumerableResponse<DicomDataset>(
             response,
             DeserializeAsAsyncEnumerable<DicomDataset>(response.Content));
     }
 
-    private async Task<DicomWebResponse> PostRequest<TContent>(
+    private async Task<DicomWebResponse> Request<TContent>(
         Uri uri,
         TContent requestContent,
+        HttpMethod httpMethod = null,
         CancellationToken cancellationToken = default) where TContent : class
     {
         EnsureArg.IsNotNull(requestContent, nameof(requestContent));
 
-        JsonSerializerOptions serializerOptions = new JsonSerializerOptions();
+        var serializerOptions = new JsonSerializerOptions();
```

Comment:
nit: Why changing these?

Reply:
It could have the type right?

Reply:
went by VS suggestion. I can revert this.